wca-regulations.md

```diff
@@ -231,11 +231,13 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
 - 9b) The official events of the WCA are:
     - 9b1) 3x3x3 Cube, 2x2x2 Cube, 4x4x4 Cube, 5x5x5 Cube, 3x3x3 One-Handed, 3x3x3 With Feet, Clock, Megaminx, Pyraminx, Skewb, and Square-1.
         - 9b1a) Round format for these events is: "Average of 5". For a Combined Round, the round format is "Best of 2" for the first phase and "Average of 5" for the second phase. Exception: In special cases, the round format for these events can be "Best of X" (where X is 1, 2, or 3), at the discretion of the WCA Board.
-    - 9b2) 3x3x3 Fewest Moves, 6x6x6 Cube, and 7x7x7 Cube.
-        - 9b2a) Round formats for these events are: "Best of X" (where X is 1 or 2) and "Mean of 3".
+    - 9b2) 6x6x6 Cube and 7x7x7 Cube.
+        - 9b2a) Round formats for these events is: "Mean of 3".
```

Comment:
I'm confused. What about combined best1/mean3?

Reply:
Hmm, now I'm confused, too. The explicit specification of combined rounds in 9b1a imply to me that combined rounds are not allowed for 6x6x6 and 7x7x7 right now. 😳

Reply:
```suggestion
        - 9b2a) Round formats for these events is: "Mean of 3". For a Combined Round, the round format is "Best of 1" for the first phase and "Mean of 3" for the second phase.
```

Let's just mimic 9b1a) here.

Reply:
I agree, seems like the best solution to remove "standalone" best of X while still allowing combined rounds (which we definitely want to keep !).